integration-snippets/controller-init.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+# This script checks if upgrade is needed and returns 0 if no upgrade
+# is needed. If upgrade is needed then check if it is in progress and
```

Comment:
This script no longer checks if there is an upgrade in progress. We either need to remove this in the comments or add the checks for progress to make it consistent.